tenderly/actions/src/index.ts

```diff
@@ -138,6 +138,7 @@ export const accountsFn: ActionFn = async (
         'x-auth-token': dataServiceAuthToken
       },
       body: JSON.stringify({
+        network: context.metadata.getNetwork(),
```

Comment:
I guess in the future this assumes that our network string will always match their network string? I think that is fine but there are some cases like ethers where mainnet is returned as "homstead" for the network name. Perhaps that will not be the case here.

Reply:
![image](https://github.com/notional-finance/notional-monorepo/assets/8759971/0b98fb4d-df39-4b35-b149-a1edbc11fc7a)
this is the `Network` type returned by `getNetwork`